codecov/__init__.py

```diff
@@ -9,6 +9,10 @@
 from time import sleep
 from json import loads
 
+from .__version__ import __title__, __description__, __url__, __version__
+from .__version__ import __author__, __author_email__, __license__
+from .__version__ import __copyright__
```

Comment:
prefer

```suggestion
from .__version__ import (
    __author__,
    __author_email__,
    __copyright__,
    __description__,
    __license__,
    __title__,
    __url__,
    __version__,
)
```

don't LOVE that we do an import from `.__version__`

Reply:
Why? That's the whole goal, so we have a single place for both pip and the package.

Reply:
I take that back. Absolute imports are preferred, but PEP-8 says 
```However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:```